### ISSUE TITLE
A seperate Colab notebook for non-Colab pro users

### ISSUE THREAD

**Issue by Vwood-png** [TheLastBen/fast-stable-diffusion#2123]
I use Colab a lot for Stable Diffusion because I have weak GPU and I honestly think that we need a Notebook that is actually usable in "Trial" of Colab. You don't need to update it like a madman, just update it if Colab ever changes something on their servers/notebooks. I honestly don't even know why we need to change the Notebook to be only usable for Pro users even though we have a normal working notebook last time (unless this guy updates it again and breaks the whole notebook, again). Also, now that I think about it, why do you update it every 2-5 days?

**Comment by DanoGL** [comment]
Yeah... about that, Free Colab is the one that stops the notebook from running, it´s a deliberate decision from Google themselves

**Comment by Gamemaster2930** [comment]
See this tweet: https://twitter.com/thechrisperry/status/1649189902079381505

Google doesn't want us free CoLab users to run SD webui anymore, because more and more poeple started using CoLab for it. And when the electricity bill came, it was huge. I doubt that Google doesn't have enough money to pay it, but on the other hand I bet it was lots of money. So they said that running SD webui in CoLab won't be allowed unless you purchase a subscription. Sucks eh? I think that even if TheLastBen made his notebook running again for free users, Google will find another way to ban users from using it. Same counts with any other notebooks out there that found a way around CoLab's "illegal code" message. Google will ban them one day the same way they banned this notebook (constant disconnection). That's why I moved from CoLab to Paperspace. TheLastBan also made a notebook for it. But Paperspace is not the only free service where you can run SD webui without being afraid of getting your account blocked from using it. TheLastBen for example made a notebook for Runpod too. But I think this one is paid. Long comment eh? I hope I made it clear, why you can't use CoLab with SD webui anymore.

P.S.: TheLastBen updated his notebook quite often these days because Google kept changing Python version etc. constantly breaking this notebook.

**Comment by Vwood-png** [comment]
@Gamemaster2930 So it is Google that makes him update the notebook so often, that clears up most of my questions then. I guess I never really thought we're that many that uses the notebook for free. Maybe because I also use Colab to render Blender projects which I don't think a lot of people do. Thanks for the info, I just wish he replied "Yeah Google changed something again" to the issues I saw last time or something like that so I know who to blame.
